### ISSUE TITLE
Pipeline editor not cleaning up validation highlighting

### ISSUE THREAD

**Issue by rsizonenko** [QualiMaster/QM-IConf#70]
Clean up only after the editor is restarted. Does not take incremental improvement of validation into account.

**Comment by Nowatzki** [comment]
Can you please check again and close if solved.
Tested several cases and it works fine since Sascha added the call of my method "resetDiagramMarkings()".

**Comment by eichelbe** [comment]
The point is that the data comes constantly in and re-triggers changes rather frequently. The initial implementation cannot cope with that as it locks the application. However, I'm happy that we have the actual one. I will test... and thanks for your work over the weekend ;)

**Comment by rsizonenko** [comment]
I still had this problem for Pipeline Editor. But the markings in other editors are reseting. Used the video scenarios for testing (could be this is a specific case).

**Comment by rsizonenko** [comment]
Also this could be not only the markings issue, but how element views are refreshed in general. Examples:
If you deselect Algorithm in the Algorithm Family members and at the same time in the Element TreeView Same Algorithm Family is expanded (opened) with Algorithm shown, this Algorithm will not disappear from the Element TreeView.
In the other direction - if you add a new Algorithm Algorithm Family Editor should be closed and opened before this new Algorithm is seen in the members list.

**Comment by eichelbe** [comment]
This is a different issue for a different editor. Please file it as an individual issue for the May release. Not relevant for now to fix.

**Comment by Nowatzki** [comment]
I checked the respective scenario you mentioned Roman and it is working for me. I add a new FamilyElement, set the algo. family to FMismatched and connect this family to datasource and Preprocessor. -> ERRORS. If I remove the newly added familyElement there are no reasoning errors and all the markings in the pipeline disappear. Are you testing another scenario??? Maybe its a very special case and we should open a new ticket for it. The general cases I know about are working for me. Sorry if I am misunderstanding something...

**Comment by rsizonenko** [comment]
You are right, it is working. Marking in PipelineEditor disappear if input/output fields of Algorithm Family are configured correctly. Members can stay unconfigured. Only Algorithm Family in Element TreeView is marked. 
Issue can be closed.
